### PR TITLE
fix: error when only selecting info on update and create operations

### PR DESCRIPTION
**Pull request by danstarns**
# Issue

#567

### CODE REVIEW

File: packages/graphql/src/translate/translate-create.ts [hunk 1]
```diff
@@ -26,19 +26,18 @@ import createConnectionAndParams from "./connection/create-connection-and-params
...
 
-    // Due to potential aliasing of returned object in response we look through fields of CreateMutationResponse
-    // and find field where field.name ~ node.name which exists by construction
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const { fieldsByTypeName } = Object.values(
-        resolveTree.fieldsByTypeName[`Create${node.getPlural({ camelCase: false })}MutationResponse`]
-    ).find((field) => field.name === node.getPlural({ camelCase: true }))!;
+    const nodeProjection = Object.values(mutationResponse).find(
+        (field) => field.name === node.getPlural({ camelCase: true })
+    );
```
Comment (angrykoala) [inline packages/graphql/src/translate/translate-create.ts:40]: :clap:

File: packages/graphql/src/translate/translate-create.ts [hunk 2]
```diff
@@ -68,57 +67,78 @@ function translateCreate({ context, node }: { context: Context; node: Node }): [
...
-                field: connectionField,
-                context,
-                nodeVariable: "REPLACE_ME",
-            });
-            connectionStrs.push(connection[0]);
-            if (!connectionParams) connectionParams = {};
-            connectionParams = { ...connectionParams, ...connection[1] };
+    let replacedProjectionParams: Record<string, unknown> = {};
+    let projectionStr: string | undefined = undefined;
+    let authCalls: string | undefined = undefined;
```
Comment (angrykoala) [inline packages/graphql/src/translate/translate-create.ts:72]: I think there is no need to assign these as are `undefined` by default

```suggestion
    let projectionStr: string | undefined;
    let authCalls: string | undefined;
```

File: packages/graphql/tests/integration/issues/only-info.test.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+ */
+
+import { Driver } from "neo4j-driver";
+import { graphql } from "graphql";
+import { gql } from "apollo-server";
+import { generate } from "randomstring";
+import neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src/classes";
+
+describe("only-info", () => {
```
Comment (angrykoala) [inline packages/graphql/tests/integration/issues/only-info.test.ts:27]: We may want to have a few more details in the description for the future :sweat_smile:

**Review by angrykoala — DISMISSED** [review]
Looks good to me!, just a couple of proposals

File: packages/graphql/tests/integration/issues/only-info.test.ts (hunk 3 above)
Comment (danstarns) [reply packages/graphql/tests/integration/issues/only-info.test.ts]: Sure

File: packages/graphql/src/translate/translate-create.ts (hunk 2 above)
Comment (danstarns) [reply packages/graphql/src/translate/translate-create.ts]: Good call

**Pull request merged**